library/billing.inc

```diff
@@ -232,8 +232,8 @@ function updateClaim(
         $version = sqlQuery("SELECT IFNULL(MAX(version),0) + 1 AS increment FROM claims WHERE patient_id = ? AND encounter_id = ?", array( $patient_id, $encounter_id ));
         if ($crossover<>1) {
```

Comment:
this is a good place to learn the $sqlBindArray technique. Above this if do:
``````
$sqlBindArray = array();
``````
Then you can support binding in these two different situations (the if and the else), by pushing the pertinent variables into the array.
Then do:
``````
sqlStatement($sql, $sqlBindArray);